### ISSUE TITLE
Proposal: Discord for org chat

### ISSUE THREAD

**Issue by gtca** [scverse/governance#21]
This issue brings [Discord](https://discord.com/) forward as a solution for the community chat platform.
This follows discussions in #9, #15, #16, #20.

Among organisations, projects and communities that have been using Discord are
- [Scala](https://www.scala-lang.org/blog/2021/12/21/discord.html) [^1]
- [Open Problems](https://openproblems.bio/)
- [Python community](https://discord.com/invite/python) — more than 300 thousand server members
- [Polars](https://github.com/pola-rs/polars)
- [students](https://discord.com/college) for educational purposes — I expect that's some part of our userbase
- multiple communities around interests such as [Machine Learning](https://discord.com/invite/cDkqDrmqFC) or around places like [Heidelberg](https://discord.gg/N8tBBPhjZq)

### Pros

- **Familiarity** for most users: it has 350 million registered users [apparently](https://en.wikipedia.org/wiki/Discord_(software))
- Rich **moderation** capabilities with user groups, possibility to mute or ban users
- Individual channels can be **customised** (e.g. read/write rights for different user groups, read-only channels for announcements, etc.)
- An option to set up a list of **guidelines** that the user has to agree to prior to joining the server
- A reasonably well **UI / UX**, when compared to solutions suited to more tech savvy users
- Options to host audio or video calls [with moderation functionality](https://discord.com/stages)
- Scalability to large community sizes
- It's _snappiness_ (other proposals have this trait in their cons)
- Apps for many **platforms**, arguably well-built, while also working in the web like other proposals

### Cons

- Proprietary [^2] 
- Some people raised concerns that using Discord might be an issue due to people with gaming profiles there making it too informal but it didn't seem to be a problem in practice.

In summary, due to the amount of features around building a community this might be a great candidate if this is going to be the main purpose of the communication platform we're choosing.

[^1]: Worth a read to see how they migrated from Gitter: https://www.scala-lang.org/blog/2021/12/21/discord.html
[^2]: https://news.ycombinator.com/item?id=29712098

**Comment by ivirshup** [comment]
Thanks for the write up! I think discord is a solid second place for me right now. I do have some concerns though

*Concerns*

* They are venture backed and not profitable yet. They may change their business model in unexpected ways.
* Has ads. I got asked to use PSN and buy some discord premium thing.
* [Extra paid features](https://support.discord.com/hc/en-us/articles/360028038352)
    * Features for threads are behind a paywall, but it’s a user paywall that then applies to a server?
* We are not the users they are focussing on. They are focussed on gaming. If they change their model or drop features, it won’t be with us in mind.
* Vibe. I think the space should keep a more professional vibe, but discord does not have this.

*Question*

* Can I choose be logged in to some subset of servers at a time? I could only find “leave server” options. Basically, I don't want social and work chats open at the same time.
* Is there a read only view for non-logged in users?
* How important is integrated voice/ video chat?

**Comment by gtca** [comment]
Just wanted to clarify a few bits, also for myself...

> Has ads. I got asked to use PSN and buy some discord premium thing.

I don't think there are ad placements in Discord though. This sounds like a banner with [their new integration](https://discord.com/blog/playstation-discord-account-connection-linking-game-status) that you get once when you update the app.

> [Extra paid features](https://support.discord.com/hc/en-us/articles/360028038352)

As I understand with Server Boosts, people can upgrade the server if that's needed. I don't think that matters for our purposes at all though?

> We are not the users they are focussing on. They are focussed on gaming. 

Historically but now there are building for communities in general as I can judge from [e.g. this](https://www.cnbc.com/2021/05/08/what-is-discord-chat-service-fosters-community-expands-beyond-gaming.html) or, more recently, [this](techcrunch.com/2022/02/04/discord-forums-mod-tools-homepages-experimental-features), or their motto on the homepage.

> I think the space should keep a more professional vibe

I was wondering about this if OpenProblems or other attempts to use Discord had any issues about it?
At least for education-focused groups, that doesn't seem to be an issue, same for some large communities that I found for the original post.

> Can I choose be logged in to some subset of servers at a time?

I think not really, seems to be only solvable through multiple accounts. But doesn't the same problem apply to Slack when one is logged into multiple workspaces on a device? The difference I see there is that one can go into each workspace to adjust the status. But here the private messages are across servers so the way it works makes sense to me.

**Comment by adamgayoso** [comment]
Closing based on agreement to use Zulip
